jetson/percep/pcl.cpp

```diff
@@ -13,7 +13,7 @@ void PassThroughFilter(pcl::PointCloud<pcl::PointXYZRGB>::Ptr & pt_cloud_ptr) {
     pass.setInputCloud(pt_cloud_ptr);
     pass.setFilterFieldName("z");
     //The z values for depth are in mm
-    pass.setFilterLimits(0.0,5000.0);
+    pass.setFilterLimits(0.0,9000.0);
```

Comment:
NIT: I'd like to see stuff like this end up in a config file

Reply:
Noted